src/pages/Explorer/Components/Actions.svelte

```diff
@@ -56,16 +58,25 @@
 
     history.push(getItemRoute(item, type), { openComments: true })
   }
+
+  function onDelete(e) {
+    e.preventDefault()
```

Comment:
`[D-0-2]` Use the [`|preventDefault`](https://svelte.dev/docs#template-syntax-element-directives-on-eventname) modifier